metricbeat/module/nginx/stubstatus/fields.yml

```diff
@@ -0,0 +1,45 @@
+- name: nginx-stubstatus
+  type: group
+  description: >
+    `nginx-stubstatus` contains the metrics that were scraped from the ngx_http_stub_status_module status page.
+  fields:
+    - name: hostname
+      type: keyword
+      description: >
+        Nginx hostname
+    - name: active
+      type: integer
+      description: >
+        The current number of active client connections including Waiting connections.
+    - name: accepts
+      type: integer
+      description: >
+        The total number of accepted client connections.
+    - name: handled
+      type: integer
+      description: >
+        The total number of handled client connections.
+    - name: dropped
+      type: integer
+      description: >
+        The total number of dropped client connections.
+    - name: requests
+      type: integer
```

Comment:
Good point, was not thinking of the implementation itself. The question here is what nginx will report as a max value and if it will exceed int or not. We hit some issues here recently in topbeat and filebeat where we use `integer` and switched to `long` as the values got too big.